foundation/foundation-mda/src/main/resources/templates/deployment/argocd/spark-operator.yaml.vm

```diff
@@ -21,7 +21,6 @@ spec:
         - {{ . }}
         {{- end }}    
     {{ end }}
-  {{- with .Values.spec.syncPolicy }}
```

Comment:
I: Spurious `with` block that had no closing `end` tag and caused helm templating errors